# Translation of Odoo Server.
# This file contains the translation of the following modules:
# 	* sign_itsme
# 
# Translators:
# Jakub Smolka, 2023
# Wil Odoo, 2023
# Tomáš Píšek, 2023
# 
msgid ""
msgstr ""
"Project-Id-Version: Odoo Server 17.0+e\n"
"Report-Msgid-Bugs-To: \n"
"POT-Creation-Date: 2023-10-26 22:21+0000\n"
"PO-Revision-Date: 2023-10-26 23:10+0000\n"
"Last-Translator: Tomáš Píšek, 2023\n"
"Language-Team: Czech (https://app.transifex.com/odoo/teams/41243/cs/)\n"
"MIME-Version: 1.0\n"
"Content-Type: text/plain; charset=UTF-8\n"
"Content-Transfer-Encoding: \n"
"Language: cs\n"
"Plural-Forms: nplurals=4; plural=(n == 1 && n % 1 == 0) ? 0 : (n >= 2 && n <= 4 && n % 1 == 0) ? 1: (n % 1 != 0 ) ? 2 : 3;\n"

#. module: sign_itsme
#: model_terms:ir.ui.view,arch_db:sign_itsme.sign_request_logs_user
msgid ""
"<small>Name: The signatory has provided this identity through itsme®</small>"
msgstr "<small>Jméno: Podepisující doplnil identitu pomocí itsme®</small>"

#. module: sign_itsme
#: model:sign.item.role,name:sign_itsme.sign_item_role_itsme_customer
msgid "Customer (identified with itsme®)"
msgstr "Zákazník (identifikován pomocí itsme®)"

#. module: sign_itsme
#. odoo-javascript
#: code:addons/sign_itsme/static/src/components/document_signable.js:0
#, python-format
msgid "Error"
msgstr "Chyba"

#. module: sign_itsme
#: model:ir.model.fields,field_description:sign_itsme.field_sign_item_role__auth_method
msgid "Extra Authentication Step"
msgstr "Dodatečný autentikační krok"

#. module: sign_itsme
#: model:ir.model.fields,help:sign_itsme.field_sign_item_role__auth_method
msgid "Force the signatory to identify using a second authentication method"
msgstr ""
"Vynuťte, aby se podespisující identifikovali s použitím další metody "
"autentikace"

#. module: sign_itsme
#. odoo-javascript
#: code:addons/sign_itsme/static/src/dialogs/itsme_dialog.xml:0
#, python-format
msgid "Go Back"
msgstr "Zpět"

#. module: sign_itsme
#. odoo-javascript
#: code:addons/sign_itsme/static/src/components/document_signable.js:0
#, python-format
msgid "Identification refused"
msgstr "Identifikace zamítnuta"

#. module: sign_itsme
#. odoo-javascript
#: code:addons/sign_itsme/static/src/dialogs/itsme_dialog.xml:0
#, python-format
msgid "Identify with itsme"
msgstr "Identifikujte se s itsme"

#. module: sign_itsme
#: model_terms:ir.ui.view,arch_db:sign_itsme.sign_request_logs_user
msgid "Name"
msgstr "Název"

#. module: sign_itsme
#. odoo-javascript
#: code:addons/sign_itsme/static/src/dialogs/itsme_dialog.xml:0
#, python-format
msgid "Please confirm your identity to finalize your signature."
msgstr "Pro dokončení podpisu, prosím, potvrďte svou identitu."

#. module: sign_itsme
#. odoo-python
#: code:addons/sign_itsme/models/sign_request_item.py:0
#, python-format
msgid "Sign request item is not validated yet."
msgstr "Položka požadavku podpisu zatím nebyla ověřena."

#. module: sign_itsme
#: model:ir.model,name:sign_itsme.model_sign_item_role
msgid "Signature Item Party"
msgstr "Role položky podpisu"

#. module: sign_itsme
#: model:ir.model,name:sign_itsme.model_sign_request
msgid "Signature Request"
msgstr "Žádost o podpis"

#. module: sign_itsme
#: model:ir.model,name:sign_itsme.model_sign_request_item
msgid "Signature Request Item"
msgstr "Položka žádosti o podpis"

#. module: sign_itsme
#. odoo-javascript
#: code:addons/sign_itsme/static/src/components/document_signable.js:0
#, python-format
msgid ""
"The itsme® identification data could not be forwarded to Odoo, the signature"
" could not be saved."
msgstr ""
"Identifikační data itsme® nemohla být předána do Odoo, podpis nemohl být "
"uložen."

#. module: sign_itsme
#: model:ir.model.fields.selection,name:sign_itsme.selection__sign_item_role__auth_method__itsme
msgid "Via itsme®"
msgstr "Pomocí itsme®"

#. module: sign_itsme
#. odoo-javascript
#: code:addons/sign_itsme/static/src/components/document_signable.js:0
#, python-format
msgid ""
"You have rejected the identification request or took too long to process it."
" You can try again to finalize your signature."
msgstr ""
"Odmítli jste požadavek na identifikaci nebo to trvalo příliš dlouho. Abyste "
"podpis dokončili, můžete to zkusit znovu."

#. module: sign_itsme
#. odoo-python
#: code:addons/sign_itsme/controllers/main.py:0
#, python-format
msgid "itsme® IAP service could not be found."
msgstr "itsme® IAP službu nelze najít."

#. module: sign_itsme
#: model:ir.model.fields,field_description:sign_itsme.field_sign_request_item__itsme_signer_birthdate
msgid "itsme® Signer's Birthdate"
msgstr "itsme® Datum narození podepisujícího"

#. module: sign_itsme
#: model:ir.model.fields,field_description:sign_itsme.field_sign_request_item__itsme_signer_name
msgid "itsme® Signer's Name"
msgstr "itsme® Jméno podepisujícího"

#. module: sign_itsme
#: model:ir.model.fields,field_description:sign_itsme.field_sign_request_item__itsme_validation_hash
msgid "itsme® Validation Token"
msgstr "itsme® Ověřovací token"
